cleanlab/internal/token_classification_utils.py

```diff
@@ -73,64 +70,80 @@ def process_token(token: str, replace: list = [("#", "")]) -> str:
         token: str
             token which potentially contains special characters
 
-        replace: list
+        replace: List[Tuple[str, str]]
             list of tuples `(s1, s2)`, where all occurances of s1 are replaced by s2
```

Comment:
Save for future PR where you clean up the docstrings and their formatting (eg. capitalization), but each of these docstrings should have some example of what could possibly be passed in here for this argument, otherwise it remains too vague for users.